x/logic/predicate/address.go

```diff
@@ -0,0 +1,69 @@
+package predicate
+
+import (
+	"context"
+	"fmt"
+
+	bech322 "github.com/cosmos/cosmos-sdk/types/bech32"
+	"github.com/ichiban/prolog/engine"
+	"github.com/okp4/okp4d/x/logic/util"
+)
+
+func Bech32Address(vm *engine.VM, address, bech32 engine.Term, cont engine.Cont, env *engine.Env) *engine.Promise {
```

Comment:
Please provide an exhautive description of the function (see: [BankBalances](https://github.com/okp4/okp4d/blob/main/x/logic/predicate/bank.go#L13-L31)).

Reply:
Oups, I forgot write it 🫢.